spec/features/observer_spec.rb

```diff
@@ -77,6 +77,22 @@
     expect(submit_button).to_not be_disabled
   end
 
+  it "observer can delete themselves as observer" do
+    proposal = FactoryGirl.create(:proposal)
+    observer = FactoryGirl.create(:user, client_slug: nil)
+
+    login_as(proposal.requester)
+    visit "/proposals/#{proposal.id}"
+    select observer.email_address, from: 'observation_user_email_address'
+    click_on 'Add an Observer'
+
+    login_as(observer)
+    visit "/proposals/#{proposal.id}"
+    delete_button = find('table.observers .button_to input[value="Delete"]')
+    delete_button.click
+    expect(page).to have_content("Deleted Observation for ")
```

Comment:
nice feature-level spec! hoping we can move this project to where all feature specs are testing at this level rather than inspecting the db or other backend stuff in expectations